src/main/java/com/navercorp/openwhisk/intellij/wskdeploy/toolwindow/action/CreateManifestTemplateAction.java

```diff
@@ -57,54 +54,35 @@ public void actionPerformed(@NotNull AnActionEvent e) {
-                .filter(file -> filterByPath(file, FILTER_PATH))
-                .collect(Collectors.toList());
-
-        if ((new WskDeployTempleteDialog(project, manifestFiles, sampleFiles)).showAndGet()) {
+        if ((new WskDeployTempleteDialog(project, META_FILES, SOURCE_FILES)).showAndGet()) {
             ApplicationManager.getApplication().runWriteAction(() -> {
-                try {
-                    if (manifestFiles.size() == 0) {
-                        // manifest template file
-                        VirtualFile manifestFile = VfsUtil.findFileByURL(ResourceUtil.getResource(getClass(), "/template/", "manifest.yaml"));
-                        final VirtualFile baseParent = VfsUtil.findFileByURL(Paths.get(basePath).toUri().toURL());
-
-                        VfsUtilCore.copyFile(this, manifestFile, baseParent);
-                        NOTIFIER.notify(project, manifestFile.getName() + " is created", NotificationType.INFORMATION);
-                    }
-
-                    if (sampleFiles.size() == 0) {
-                        // action template file
-                        VirtualFile sampleFile = VfsUtil.findFileByURL(ResourceUtil.getResource(getClass(), "/template/src/", "hello.js"));
-                        final VirtualFile sampleParent = VfsUtil.createDirectories(basePath + "/src");
-                        VfsUtilCore.copyFile(this, sampleFile, sampleParent);
-                        NOTIFIER.notify(project, sampleFile.getName() + " is created", NotificationType.INFORMATION);
-                    }
+                for (String filePath : META_FILES) {
+                    copyFile(project, "/template", filePath, basePath);
+                }
 
-                    // refresh manifest file tree
-                    EventUtils.publish(project, RefreshWskDeployManifestListener.TOPIC, RefreshWskDeployManifestListener::refreshWskDeployManifest);
-                } catch (IOException e1) {
-                    LOG.error(e1);
+                for (String filePath : SOURCE_FILES) {
+                    copyFile(project, "/template/src", filePath, basePath + "/src");
                 }
+
+                // refresh manifest file tree
+                EventUtils.publish(project, RefreshWskDeployManifestListener.TOPIC, RefreshWskDeployManifestListener::refreshWskDeployManifest);
             });
         }
     }
 
-    private boolean filterByPath(PsiFile file, String[] filterList) {
-        for (String path : filterList) {
-            if (file.getVirtualFile().getPath().contains(path)) {
-                return false;
+    private void copyFile(Project project, String resourcePath, String fileName, String toDir) {
+        try {
+            boolean exist = Arrays.stream(FilenameIndex.getFilesByName(project, fileName, GlobalSearchScope.allScope(project)))
+                    .anyMatch(file -> file.getVirtualFile().getPath().startsWith(toDir));
+
+            if (!exist) {
```

Comment:
If there are already some files there, how does the plugin inform users?

Reply:
Good idea. I have added notification for it